images/tier3/inprogress-circle.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="30" height="30" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I think an argument can be made that this icon should be mirrored in RTL mode by adding the `mirror-in-rtl="true"` attribute to the `svg` element.

From [Google's Material Design guidelines on RTL icons](http://google.github.io/material-design-icons/#icons-in-rtl):
> When text, layout, and iconography are mirrored to support right-to-left UIs, anything that relates to time should be depicted as moving from right to left